src/DynamoCore/Search/SearchElements/ZeroTouchSearchElement.cs

```diff
@@ -13,6 +13,23 @@ public class ZeroTouchSearchElement : NodeSearchElement
     {
         private readonly FunctionDescriptor functionDescriptor;
 
+        public override string Description
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(functionDescriptor.Description))
+                    return Dynamo.UI.Configurations.NoDescriptionAvailable;
+
+                return functionDescriptor.Description;
+            }
+            set
+            {
+                if (value == functionDescriptor.Description) return;
+                description = value;
+                OnPropertyChanged("Description");
+            }
+        }
+
```

Comment:
What's the point of setting `description` data member when the `getter` always return `functionDescriptor.Description`? There seems to be some design issue here.